inc/Api/Api.php

```diff
@@ -274,6 +279,9 @@ public function save_api_information() {
 			'enable'                => $enabled,
 			'syncronize_additional' => $syncronize_additional,
 			'enable_cart'           => $cart_enabled,
+			'checkbox_display'      => $checkbox_display,
+			'checkbox_order'        => $checkbox_order,
+			'checkbox_location'     => $checkbox_location,
```

Comment:
I would combine order and location fields into a single field in the database, and the value would be full WooCommerce hook name. This way we could target even those hooks that do not follow before/after logic, and it makes the hook targeting when WooCommerce decides to change things.

What do you think?

Reply:
I kept them separate for better handling at the settings page. Then there is no need for some kind of parser when receiving or saving hook in the settings page. So the implementation side only has to parse hook details into the required form. I could do a serialized array for combining two values into a single settings field.

> This way we could target even those hooks that do not follow before/after logic

This is somewhat against the current approach to [make it stupid simple](https://images.prismic.io/netmums/60fd8dd31aede199a23501d7e25d85286392ce8f_grandma-friendly-remote-control.jpg?auto=compress,format). If we want that kind of behavior we should go with the MailChimp approach and provide a text field for the hook.

Reply:
OK. Let's keep it as is.